index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <header>
+        <nav>
+          <a class="logo" href="">My Logo</a>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="Menu">
+          </div>
+      </nav>
```

Comment:
- [ ] Great job adding the `header` tag to your project. But according to the project requirements,  the Toolbar which is the `Hamburger menu` and the `logo` are the only ones that need to be enclosed in the (`header` tags). 
- [ ] The (`section` tag) will also enclose the Headline section as you have rightly done. So all you need to do now is to enclose the nav in the header :+1: